src/qt/splashscreen.h

```diff
@@ -36,8 +36,8 @@ public Q_SLOTS:
     /** Show message and progress */
     void showMessage(const QString &message, int alignment, const QColor &color);
 
-    /** Sets the break action */
-    void setBreakAction(const std::function<void(void)> &action);
+    /** Sets if cancel is currently possible */
```

Comment:
Added some additional comments. I'm not really sure if thats a better name, though I agree cancel_possible is a bit strange.